SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -56,15 +71,15 @@ void SpatialVirtualWorkerTranslator::ApplyVirtualWorkerManagerData(Schema_Object
 // after a PiE restart.
 bool SpatialVirtualWorkerTranslator::IsValidMapping(Schema_Object* Object) const
 {
-	int32 TranslationCount = (int32)Schema_GetObjectCount(Object, SpatialConstants::VIRTUAL_WORKER_TRANSLATION_MAPPING_ID);
+	const int32 TranslationCount = Schema_GetObjectCount(Object, SpatialConstants::VIRTUAL_WORKER_TRANSLATION_MAPPING_ID);
```

Comment:
This will generate a compilation warning on some platforms, unless `Schema_GetObjectCount` changed from returning `uint32_t`?